exercises/partial_application_with_bind/problem.ko.md

```diff
@@ -0,0 +1,45 @@
+# 해야할 일
+
+**Function#bind를 사용해** 메시지에 이름공간(namespace)을 사용할 수 있는 로그 함수를 구현하세요.
```

Comment:
I didn't know that '이름공간' can be used. https://ko.wikipedia.org/wiki/이름공간
We can use '이름공간' or '네임스페이스', but I prefer '네임스페이스' since it's more widely used.